src/instruments/src/SD/Pages/Wheel/Wheel.tsx

```diff
@@ -0,0 +1,309 @@
+            <WheelArch x={406} y={370} type="bottom" />
+
+            <Gear x={401} y={210} type="right" />
+            <Wheels
+                x={436}
+                y={318}
+                left={{ number: 3, temperature: temperatures[2], hottest: maxTemperatureIndex === 2 }}
+                right={{ number: 4, temperature: temperatures[3], hottest: maxTemperatureIndex === 3 }}
+            />
+        </EcamPage>
+    );
+};
+
+const NoseWheelSteering = ({ x, y }) => {
+    const [antiSkidActive] = useSimVar('ANTISKID BRAKES ACTIVE', 'Bool', maxStaleness);
+
+    return !antiSkidActive ? (
+        <SvgGroup x={x} y={y}>
+            <HydraulicIndicator x={0} y={0} type="Y" />
+
+            <text x={29} y={17} className="big-text align-left color-amber">N/W STEERING</text>
+        </SvgGroup>
+    ) : null;
+};
+
+const AntiSkid = ({ x, y }) => {
+    const [antiSkidActive] = useSimVar('ANTISKID BRAKES ACTIVE', 'Bool', maxStaleness);
+
+    return !antiSkidActive ? (
+        <SvgGroup x={x} y={y}>
+            {/* <!-- Text --> */}
+            <text x={0} y={0} className="big-text color-amber">ANTI SKID</text>
+
+            {/* <!-- Brake and Steering Control Units --> */}
+            <path className="shape color-gray" d="m 76 5 h 20 v -25" />
+            <text className="big-text color-green" x={86} y={0}>1</text>
+
+            <path className="shape color-gray" d="m 103 5 h 20 v -25" />
+            <text className="big-text color-green" x={113} y={0}>2</text>
+        </SvgGroup>
+    ) : null;
+};
+
+const LandingGearCtl = ({ x, y }) => {
+    const [landingGearLeft] = useSimVar('GEAR LEFT POSITION', 'Percent Over 100', maxStaleness);
+    const [landingGearCenter] = useSimVar('GEAR CENTER POSITION', 'Percent Over 100', maxStaleness);
+    const [landingGearRight] = useSimVar('GEAR RIGHT POSITION', 'Percent Over 100', maxStaleness);
+
+    const landingGearInTransit = landingGearLeft > 0 && landingGearLeft < 1
+        || landingGearCenter > 0 && landingGearCenter < 1 || landingGearRight > 0 && landingGearRight < 1;
+
+    return landingGearInTransit ? (
+        <text id="center-lg-ctl" x={x} y={y} className="big-text align-left color-amber">L/G CTL</text>
+    ) : null;
+};
+
+const NormalBraking = ({ x, y }) => {
+    const hydraulics = useHydraulics();
+
+    return !hydraulics.G.available ? (
+        <SvgGroup x={x} y={y}>
+            <HydraulicIndicator x={0} y={0} type="G" />
+
+            <text x={86} y={18} className="big-text color-amber">NORM BRK</text>
+        </SvgGroup>
+    ) : null;
+};
+
+const AlternateBraking = ({ x, y }) => {
+    const hydraulics = useHydraulics();
+
+    return !hydraulics.G.available ? (
+        <SvgGroup x={x} y={y}>
+            <HydraulicIndicator x={0} y={0} type="Y" />
+
+            <text x={86} y={18} className="big-text color-green">ALTN BRK</text>
+            <AccumulatorOnly x={45} y={28} />
+        </SvgGroup>
+    ) : null;
+};
```

Comment:
Actually pressure threshold for brake system switch is different from "hyd_available" thresholds.
// Feedback by Cpt. Chaos — 25/04/2021 #pilot-feedback
    const MIN_PRESSURE_BRAKE_ALTN_HYST_LO: f64 = 1305.;
    const MIN_PRESSURE_BRAKE_ALTN_HYST_HI: f64 = 2176.;

Reply:
I would suggest creating a separate PR which modifies the WHEEL, F/CTL and HYD page to use calculated values coming from `systems.wasm` in a separate PR. This PR refactors from Asobo's JS code to our React implementation.